dipy/viz/actor.py

```diff
@@ -0,0 +1,648 @@
+    image_resliced.SetResliceTransform(transform)
+    image_resliced.AutoCropOutputOn()
+    image_resliced.SetInterpolationModeToLinear()
+    image_resliced.Update()
+
+    if nb_components == 1:
+        if lookup_colormap is None:
+            # Create a black/white lookup table.
+            lut = colormap_lookup_table((0, 255), (0, 0), (0, 0), (0, 1))
+        else:
+            lut = lookup_colormap
+
+    x1, x2, y1, y2, z1, z2 = im.GetExtent()
+    ex1, ex2, ey1, ey2, ez1, ez2 = image_resliced.GetOutput().GetExtent()
+
+    class ImageActor(vtk.vtkImageActor):
+
+        def input_connection(self, output):
+            if vtk.VTK_MAJOR_VERSION <= 5:
+                self.SetInput(output.GetOutput())
+            else:
+                self.GetMapper().SetInputConnection(output.GetOutputPort())
+            self.output = output
+            self.shape = (ex2 + 1, ey2 + 1, ez2 + 1)
+
+        def display_extent(self, x1, x2, y1, y2, z1, z2):
+            self.SetDisplayExtent(x1, x2, y1, y2, z1, z2)
+            if vtk.VTK_MAJOR_VERSION > 5:
+                self.Update()
+
+        def display(self, x=None, y=None, z=None):
+            if x is None and y is None and z is None:
+                self.display_extent(ex1, ex2, ey1, ey2, ez2/2, ez2/2)
+            if x is not None:
+                self.display_extent(x, x, ey1, ey2, ez1, ez2)
+            if y is not None:
+                self.display_extent(ex1, ex2, y, y, ez1, ez2)
+            if z is not None:
+                self.display_extent(ex1, ex2, ey1, ey2, z, z)
+
+        def opacity(self, value):
+            if vtk.VTK_MAJOR_VERSION <= 5:
+                self.SetOpacity(value)
+            else:
+                self.GetProperty().SetOpacity(value)
+
+        def copy(self):
+            im_actor = ImageActor()
+            im_actor.input_connection(self.output)
+            im_actor.SetDisplayExtent(*self.GetDisplayExtent())
+            im_actor.opacity(opacity)
+            return im_actor
+
+        def set_position(self, wx, wy, wz):
+            self.SetPosition(wx, wy, wz)
+
+        def get_position(self):
+            return self.GetPosition()
+
+    image_actor = ImageActor()
+    if nb_components == 1:
+        plane_colors = vtk.vtkImageMapToColors()
+        plane_colors.SetLookupTable(lut)
+        plane_colors.SetInputConnection(image_resliced.GetOutputPort())
+        plane_colors.Update()
+        image_actor.input_connection(plane_colors)
+    else:
+        image_actor.input_connection(image_resliced)
+    image_actor.display()
+    image_actor.opacity(opacity)
+
+    return image_actor
+
+
+def streamtube(lines, colors=None, opacity=1, linewidth=0.01, tube_sides=9,
+               lod=True, lod_points=10 ** 4, lod_points_size=3,
+               spline_subdiv=None, lookup_colormap=None):
+    """ Uses streamtubes to visualize polylines
+
+    Parameters
+    ----------
+    lines : list
+        list of N curves represented as 2D ndarrays
```

Comment:
If i remeber well, later on wthe the function "lines_to_vtk_polydata()"
points are stacked with "points_array = np.vstack(lines)"

so: this could be a List of 2D arrays [nb_lines][nb_pts_pl x 3]
or a 3D array [nb_lines x nb_pts x 3]